src/webui/webapplication.h

```diff
@@ -43,8 +43,8 @@
 #include "base/utils/net.h"
 #include "base/utils/version.h"
 
-constexpr Utils::Version<int, 3, 2> API_VERSION {2, 2, 0};
-constexpr int COMPAT_API_VERSION = 23;
+constexpr Utils::Version<int, 3, 2> API_VERSION {2, 2, 1};
+constexpr int COMPAT_API_VERSION = 24;
```

Comment:
Although I'm not sure new API methods are supported via legacy API.

Reply:
Since I am totally non-participant in WebUI stuff, please take a look at it. If it isn't used then I shouldn't bump the `COMPAT_API_VERSION`.

Reply:
IIRC, the original issue is about new RSS related method. RSS isn't available via old API.